project/app/views/index.jade

```diff
@@ -5,14 +5,29 @@ block content
     .titleImage
       img.className#IdName(src="/images/college.jpg" alt="Title image")
     .row
-      button(class="navButton" onclick="redirect('/predict')") Go to Predict
-      button(class="navButton" onclick="redirect('/about')") Go to About
-      button(onclick="redirect('/login_form')" class="navButton") Go to Login form
-
+      button(class="ui-button ui-widget ui-corner-all" onclick="redirect('/predict')") Go to Predict
+      button(class="ui-button ui-widget ui-corner-all" onclick="redirect('/about')") Go to About
+      button(class="ui-button ui-widget ui-corner-all" onclick="redirect('/login_form')" ) Go to Login form
     .row
       h1= title
       p Welcome to #{title}
-      .image-block
-        img.className#IdName(src="/images/1.gif" alt="Index page image here")
+    .row
+      h2#ui-state-default
+        Prediction Attributes Infomration
+      div#tabs
+        ul
+          li
+            a(href="gre.jade") GRE
+          li
+            a(href="gre.jade") TOEFL
+          li
+            a(href="gre.jade") SAT
+    .row
+      .col-md-6
+        .div#Drag
```

Comment:
is this a jquery-ui requirement that you use upper case `Drag` instead of `drag`? If not let's stick with lower case.